app/api/models/user.py

```diff
@@ -151,37 +151,6 @@ def add_models_to_namespace(api_namespace):
         "password": fields.String(required=True, description="User's password"),
     },
 )
-
-# TODO: Remove 'expiry' after the android app refactoring.
-login_response_body_model = Model(
-    "Login response data model",
-    {
-        "access_token": fields.String(required=True, description="User's access token"),
-        "expiry": fields.Float(
-            required=True, description="Access token expiry UNIX timestamp"
-        ),
```

Comment:
these are the only lines that should be removed in this pull request :) All the rest you removed, you should keep

Reply:
Thanks for your review :)